src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="/src/styles/index.scss"
+      href="/src/styles/index.css"
```

Comment:
The 'href' attribute should point to the correct CSS file. If 'index.css' is the compiled version of 'index.scss', then the path should be correct, but make sure that the file actually exists in the 'src/styles/' directory after compilation.